### ISSUE TITLE
Deceptive-Fives Conventions

### ISSUE THREAD

**Issue by iamwhoiamhahaha** [Zamiell/hanabi-conventions#422]
* Save the 5s with color clues
* *Loaded Play Clues* are turned on but only apply to the color clues focusing on *Chop*

**Comment by Zamiell** [comment]
yup

**Comment by Zamiell** [comment]
do you wanna do a PR

**Comment by iamwhoiamhahaha** [comment]
sure but i'm gonna propose something more

**Comment by Zamiell** [comment]
start with this, do PR, separate proposal in new thread

**Comment by iamwhoiamhahaha** [comment]
wanna do them at the same time though
i'm writing new conventions now

**Comment by iamwhoiamhahaha** [comment]
### Deceptive 5 Positional Clues
* This convention only apply to Deceptive-Fives.
* Players can perform a *Positional Clue* with a 2, 3, 4 clue by using a 5.

### Deceptive 5 Number Ejections
* This convention only apply to Deceptive-Fives.
* Similar to a 5CE. If it doesn't "solve" the problem by blind-playing the *Finesse Position* when someone give a *Position Clue*, then the next player should blind-play their *Second Finesse Position* to demonstrate the information to the whole team.

**Comment by iamwhoiamhahaha** [comment]
I'm not sure if they are good enough to define *Deceptive 5 Positional Clues* and *Deceptive 5 Number Ejections*

**Comment by iamwhoiamhahaha** [comment]
### Deceptive-Five Rank Promise
- When a card is focused by a rank clue, the number must match the true identity of the card. In other words, the focused card is *Promised* to match the number of the rank clue.
- For example, in a 5-suits game:
  - One copy of yellow 3 has discarded.
  - In Deceptive-Fives, since there is no way to distinguish a yellow 3 from a yellow 5, players are not allowed to save the yellow 5 with a 3 clue.

### Deceptive-Five Rank Saves
- If players have *Free Choice* between a rank clue and a color clue, they must save a critical 2, 3 or 4 with rank clues if it would look like a 5 by giving a color clue.
- For example, in a 5-suits game:
  - One copy of yellow 3 has discarded.
  - In Deceptive-Fives, since there is no way to distinguish a yellow 3 from a yellow 5, players are not allowed to save the yellow 3 with a yellow clue if they have *Free Choice*.
  - In other words, we save the yellow 3 with rank and save the yellow 5 with color.

### Deceptive-Five Suboptimal Color Play Clues
- If a player breaks the *Deceptive-Five Rank Saves* saving the specific critical 2, 3, 4 by giving a color clue, it is a signal as a *Play Clue*.

### Deceptive-Five Pseudo Saves
- Players are allowed to save the 5 with rank clues if the card which makes ambiguity has been played.
- For example, in a 5-suits game:
  - Yellow 3 is played on the stack. Green 3 is critical.
  - Bob has a yellow 5 on *Chop* and a green 3 which is one-away-from-chop.
  - Alice is allowed to give a 3 clue and save the yellow 5 and the green 3 at the same time.

**Comment by iamwhoiamhahaha** [comment]
### Deceptive-Five Number Ejections
- Players can break *Deceptive-Five Rank Promise* and give a rank clue by using a 5.
- In this situation, the next player has to blind-play their *Second Finesse Position* as a *Deceptive-Five Number Ejection* to deliver the information to the whole team.
- When this happens, the clue receiver should mark their card as a 5.
- For example, in a 5-suits game:
  - It is the first turn of the game and nothing is played on the stacks.
  - Alice gives a 3 clue focusing a yellow 5 in Cathy's hand.
  - If Bob blind-plays slot 1 then Cathy would mark her card as a 3, which is quite bad.
  - Thus, Bob blind-plays his *Second Finesse Position* as a *Deceptive-Five Number Ejections*. It is a yellow 1 and it successfully plays.
  - Cathy is surprised since it looks like a *3 Bluff* before Bob blind-plays. Then she realizes it is a *Deceptive-Five Number Ejection*. Therefore, she marks her card as a 5.

**Comment by timotree3** [comment]
Seems good. Although it might be good to configure the save clues the opposite way, save 5's with color and other cards with number so as to not look like 5's. That way we have fewer total save interpretations because of the overlap between 2 saves and save clues for whatever 5 looks like a 2

Or maybe we should save 5's with color, except for the 5's that receive 2 clues, which should be saved with number 2 clue?

**Comment by timotree3** [comment]
> In Deceptive-Fives, since there is no way to distinguish a yellow 3 from a yellow 5, players are not allowed to save the yellow 3 with a yellow clue if they have Free Choice.

I believe that the Yellow 5 receives 2 clues normally, not 3 clues. I think this should say Green 5 and Green 3

**Comment by iamwhoiamhahaha** [comment]
> I believe that the Yellow 5 receives 2 clues normally, not 3 clues. I think this should say Green 5 and Green 3
O true. I'll edit them

**Comment by iamwhoiamhahaha** [comment]
### Deceptive-Five Number Ejection Examples

#### Example 1

https://hanab.live/replay/356505#6

![Deceptive-Five Number Ejection 1](https://user-images.githubusercontent.com/50160641/100638877-7e670800-336f-11eb-8068-ee3d3c6d1307.png)

#### Example 2

https://hanab.live/replay/356521#16

![Deceptive-Five Number Ejection 2](https://user-images.githubusercontent.com/50160641/100639005-9f2f5d80-336f-11eb-9f0b-5adcb2810c19.png)

#### Example 3

https://hanab.live/replay/356547#7

![Deceptive-Five Number Ejection 3](https://user-images.githubusercontent.com/50160641/100639026-a5bdd500-336f-11eb-99ee-118816803370.png)

**Comment by timotree3** [comment]
In example 3 how does jeff know whether it's a Rank-Choice Ejection or a 5's Number Ejection? I still think 2 should be a save clue on teal 5 and teals should be an ejection in that example.

**Comment by iamwhoiamhahaha** [comment]
In this variant, we save 5s with color clues. If kyle does nothing, I would mark my t5 as t2 after receiving a teal clue. It's not a *Rank-Choice Ejection* because it is not a 5 clue.

**Comment by timotree3** [comment]
> It's not a *Rank-Choice Ejection* because it is not a 5 clue.
We have RCE for 2 saves as well as 5 saves

**Comment by iamwhoiamhahaha** [comment]
I mean it's not a *Rank-Choice Ejection (with a number 5)*. But I see your point. Are you saying Jeff would bomb the t5 as g2 or p2?

Well, first of all, it's not a *Rank-Choice Ejection* in this situation because kimbi doesn't have *Free Choice* between a 2 clue and a color clue (the 2 clue touches two cards). But secondly, if Jeff doesn't have t2, then it conflicts with RCE in that situation.

**Comment by timotree3** [comment]
> - When a card is focused by a rank clue, the number must match the true identity of the card. In other words, the focused card is Promised to match the number of the rank clue.

I think Deceptive-Five Rank Promise can be made much more simple:
- Players are not allowed to clue a 5 as the focus of a rank clue. If a player does clue a 5 as the focus of a rank clue, then it signals one of the special moves detailed below.

**Comment by Zamiell** [comment]
> Players are not allowed to clue a 5 as the focus of a rank clue. If a player does clue a 5 as the focus of a rank clue, then it signals one of the special moves detailed below.

To clarify, this only applies to rank save clues.
e.g. you can do the analog of the *Pink Play Clue Lie*

**Comment by Zamiell** [comment]
### Rank Save Promise

- The rank of a *Chop-Focus* number clue must match the number of the card. In other words, players are not allowed to use a rank clue to touch a 5 if it is on chop.

**Comment by Zamiell** [comment]
### Deceptive-Five Number Ejection

- If a player breaks the *Rank Save Promise*, they must be intending to send an additional message.
- In this situation, the intent is to have the next player *Eject*. When this happens, the clue receiver should mark their card as a 5.
- For example, in a 3-player game:
  - It is the first turn of the game and nothing is played on the stacks.
  - Alice gives a number 3 clue, touching a green 5 in Cathy's hand.
  - If Bob blind-plays his *Finesse Position*, then Cathy would mark her card as a 3, which would be bad.
  - Instead, Bob blind-plays his *Second Finesse Position* as a *Deceptive-Five Number Ejection*. It is a yellow 1 and it successfully plays.
  - Cathy knows that an *Ejection* happened. The only *Ejection* that matches this sequence of events is a *Deceptive-Five Number Ejection*, so she marks her 3 as a 5.

**Comment by Zamiell** [comment]
i dont like the other things you proposed, and looks like noone else cares, so i will close this for now
